neuralmonkey/runners/xent_runner.py

```diff
@@ -9,25 +9,26 @@
 from neuralmonkey.runners.base_runner import BaseRunner
 
 
-class PerplexityRunner(BaseRunner[AutoregressiveDecoder]):
+SupportedDecoders = Union[AutoregressiveDecoder]
```

Comment:
Tady neni pylint nešťastnej?

Reply:
Tady asi šlo už rovnou připsat sequence labelátor, nešlo by?

Reply:
Já nevim jak se zachová mypy, když bude generic instanciovanej s unionem, podle mě to jen tak nemůžeš.

Reply:
s/instanciovanej/parametrizovanej/

Reply:
Ale možná to jde, jak tak nad tim googlim/přemejšlim.. Od Unionu se určitě nedá dědit, ale uvnitř generiku asi bejt může.

Reply:
Viz vyse. Labelator tam mel byt, ale v dane chvili nebyly sjednocene train_xents. S rebasem ho tam pridam.